lib/scout/view/desktop_scout.dart

```diff
@@ -50,7 +50,7 @@ class _DesktopScoutState extends State<DesktopScout> {
   AthleteScoutModel? curAthlete;
   int _widgetIndex = 0;
   int _marketVsBookPriceIndex = 0;
-  EthereumChain? _selectedChain;
+  EthereumChain _selectedChain = EthereumChain.polygonMainnet;
```

Comment:
I am not sure why you need to set it to `EthereumChain.polygonMainnet`? I do not think there is any advantage of this.